build/changelog.py

```diff
@@ -86,10 +100,17 @@ def get_github_id(commit_message, commit_id, token):
 
 def mention_author(commit_message, commit_id, token):
     username = get_github_id(commit_message, commit_id, token)
-    if username:
+    if username not in org_members_usernames:
         return "authored by @[{author}](https://github.com/{author})".format(author=username)
     return ""
 
+def get_issue_reporter(issue_id, token):
+    url = "https://api.github.com/repos/open-policy-agent/opa/issues/{issue_id}".format(issue_id=issue_id)
+    issue_data = fetch(url, token)
+    username = issue_data.get("user", "").get("login", "")
+    if username not in org_members_usernames:
+        return "reported by @[{reporter}](https://github.com/{reporter})".format(reporter=username)
+    return ""
 
 def fixes_issue_id(commit_message):
     match = re.search(r"Fixes:?\s*#(\d+)", commit_message)
```

Comment:
There's a bunch of other "keywords" that works for this too, see https://docs.github.com/en/enterprise-server@3.0/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests#linking-a-pull-request-to-an-issue

I have at least seen "Closes" and "Resolves" used.. but I guess "Fixes" is the more popular option.